Applications/SlicerApp/Testing/Python/BRAINSFitRigidRegistrationCrashIssue4139.py

```diff
@@ -103,12 +103,12 @@ def test_BRAINSFitRigidRegistrationCrashIssue4139(self):
 
     logic = BRAINSFitRigidRegistrationCrashIssue4139Logic()
 
-    from SampleData import SampleDataLogic
+    import SampleData
 
-    fixed = SampleDataLogic().downloadMRBrainTumor1()
+    fixed = SampleData.downloadSample('MRBrainTumor1')[0]
```

Comment:
The [0] at the end does not look nice. What do you think about having `downloadSamples` - returning all files in the sample; and `def downloadSample(*arg): downloadSamples[0]` returning the first sample?

Reply:
Agreed.